language/compiler/ir_to_bytecode/src/compiler.rs

```diff
@@ -41,6 +41,8 @@ struct LoopInfo {
     breaks: Vec<usize>,
 }
 
+type TypeParameterMap = HashMap<String, usize>;
```

Comment:
Better name for this or is it actually good?

Reply:
Seems fine by me